### ISSUE TITLE
All pie chart data point displayed in the same color in the different contrast themes.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#19339]
#### Expected behaviour
It should be possible to set the colors in the pie chart so that each slice of the pie chart is displayed in different color when in high contrast mode. The different colors should also be visible in the legend, next to the buttons that filter the chart.

#### Actual behaviour
When we apply Aquatic contrast theme, the color of data points within the series in the pie chart seems to be overridden to always be "window". So all slices in the chart are displayed in the same color. The colors of the data points aren't visible in the legend at all.

#### Live demo with steps to reproduce

Pre-requisite:
1. Enable contrast theme such as Aquatic, Desert, Dusk and Night Sky using below steps:
2. Open "Settings"
3. TAB to "Accessibility" and select it using ENTER key
4. TAB to "Contrast theme" and select it using ENTER key, then TAB to "Contrast theme" dropdown, open it using ALT+DOWN arrow key, select "Aquatic" contrast theme.

Repro-steps:
1. Launch the application using URL: https://www.highcharts.com/demo/pie-legend
2. "HighCharts" screen should open.
3. TAB to "Highcharts Demos › Pie with legend" section where pie chart is present.
4. Observe that all pie chart data point displayed in the same color in the different contrast themes.

#### Product version

Chromium Edge: Version 112.0.1722.17 (Official build) dev (64-bit)
Application: HighChartS

**Comment by oysteinmoseng** [comment]
Closing as duplicate of https://github.com/highcharts/highcharts/issues/15921

**Comment by oysteinmoseng** [comment]
@msftedad Closed this as duplicate, but we will prioritize this functionality in the near future.
